### PR TITLE
#jurc413q updating admin panel

### PR DESCRIPTION
**Pull request by Eli250**
#### What does this PR do?

Update admin panel

#### Description of Task to be completed?

This PR adds features to admin pane that were not present before.
#### How should this be manually tested?

This should be tested by checking if admin can view all created posts and see the total counts on dashboard.

#### What are the relevant pivotal tracker stories?

ZNbdHQuX

#### Screenshots:

Screenshot from 2021-12-28 20-36-21
Screenshot from 2021-12-28 20-36-26

#### Questions
N/A
:

### CODE REVIEW

File: UI/script.js [hunk 1]
```diff
@@ -1,27 +1,28 @@
...
+      $('.scroll-up-btn').removeClass('show');
+    }
+  });
+  // slide-up script
+  $('.scroll-up-btn').click(function () {
+    $('html').animate({ scrollTop: 0 });
+    // removing smooth scroll on slide-up button click
+    $('html').css('scrollBehavior', 'auto');
+  });
+  // Toggle menu / navbar script
```
Comment (ncutixavier) [inline UI/script.js:22]: Remove Jquery in your code

File: UI/script.js [hunk 2]
```diff
@@ -1,27 +1,28 @@
...
 
-});
\ No newline at end of file
+  $('.menu-btn').click(function () {
+    $('.navbar .menu').toggleClass('active');
+    $('.menu-btn i').toggleClass('active');
+  });
+});
```
Comment (ncutixavier) [inline UI/script.js:28]: Remove Jquery

**Review by ncutixavier — CHANGES_REQUESTED** [review]

File: UI/script.js (hunk 2 above)
Comment (ncutixavier) [reply UI/script.js:28]: Change your PR title to follow naming convention

**Review by Eli250 — COMMENTED** [review]
I have removed all jQuery files and replaced with pure javaScript.
You can review them in my recent push.

###Screenshots
Screenshot from 2021-12-30 13-52-48
Screenshot from 2021-12-30 13-52-32

**Review by ncutixavier — APPROVED** [review]

**Pull request merged**